src/index.js

```diff
@@ -2,10 +2,29 @@ import React, { useEffect, useState } from 'react';
 import ReactDOM from 'react-dom';
 import { withRouter } from 'react-router-dom';
 import './index.css';
+import { cook } from './Components/Navbar/UserNavbar';
 
 import Routing from './Routing';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import { checkIfUserIsSignedIn } from './APIFunctions/Auth';
+import { ColorTypes } from 'pdf-lib';
+import { create } from 'react-filepond';
+import { setFetchMethod } from 'darkreader';
+
+import {
+  enable as enableDarkMode,
+  disable as disableDarkMode,
+  auto as followSystemColorScheme,
+} from 'darkreader';
+
+setFetchMethod(window.fetch);
+enableDarkMode({
```

Comment:
do we need `21-27`?

Reply:
21 - 27 is part of the import for the darkreader